dist/google-music.js

```diff
@@ -279,6 +293,16 @@ proto.rating = {
     if (el && !this.rating._isElSelected(el)) {
       el.click();
     }
+  },
+
+  // Reset the rating
```

Comment:
Ah, I guess I went comment crazy in this chunk. It's weird since the repeat ones don't have it. Yea, feel free to keep the comment here. Thanks for being consistent and keeping me in check =)